InterviewFlashCards-Bridging-Header.h

```diff
@@ -0,0 +1,16 @@
+//
+//  InterviewFlashCards-Bridging-Header.h
+//  
+//
+//  Created by Charles Kang on 10/14/16.
```

Comment:
We should probably remove the comments here that say Charles Kang created it... And it applies to all the classes. Unless we want to keep them, @vhart what are your thoughts?